src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListener.java

```diff
@@ -154,6 +158,26 @@ public void onCompleted(Run run, @Nonnull TaskListener listener) {
             Map<String, Set<String>> tags = buildData.getTags();
             String hostname = buildData.getHostname("unknown");
             client.gauge("jenkins.job.duration", buildData.getDuration(0L) / 1000, hostname, tags);
+            logger.fine(String.format("[%s]: Duration: %s", buildData.getJobName(null), toTimeString(buildData.getDuration(0L))));
+
+            if (run instanceof WorkflowRun) {
+                RunExt extRun = RunExt.create((WorkflowRun) run);
+                long pauseduration = 0;
+                for (StageNodeExt stage : extRun.getStages()) {
+                    pauseduration += stage.getPauseDurationMillis();
```

Comment:
We could also get the checkoutduratio of the SCM Checkout stage. This would show how long the checkout of the code took. I can also include it here. Was not sure if I should a further PR afterwards for it.